modules/aws/rds/main.tf

```diff
@@ -46,47 +46,69 @@ module "db" {
   version = "v3.3.0"
 
   identifier = var.identifier
-
+  /* Subnets */
+  create_db_subnet_group                = var.create_db_subnet_group
+  db_subnet_group_name                  = var.db_subnet_group_name
+  db_subnet_group_description           = var.db_subnet_group_name
+  db_subnet_group_use_name_prefix       = var.db_subnet_group_use_name_prefix
+  subnet_ids = var.vpc.database_subnets
+  
+  vpc_security_group_ids = [module.security_group.security_group_id]
+  
   name                                  = var.name
-  allocated_storage                     = var.storage
   engine                                = var.engine
   engine_version                        = var.engine_version
-  major_engine_version                  = var.major_engine_version
-  family                                = var.family
-
-  instance_class                        = var.instance_class
-  username                              = var.username
-  password                              = var.password
-
-  snapshot_identifier                   = var.snapshot_identifier
-
-  subnet_ids = var.vpc.database_subnets
-  vpc_security_group_ids = [module.security_group.security_group_id]
-
-  apply_immediately                     = true
-  skip_final_snapshot                   = false
-  auto_minor_version_upgrade            = true
-  backup_retention_period               = 7
-  backup_window                         = "02:21-02:51"
-  copy_tags_to_snapshot                 = true
-  delete_automated_backups              = false
-  deletion_protection                   = false
-  iam_database_authentication_enabled   = false
-  license_model                         = var.engine == "postgres" ? "postgresql-license" : ""
-  maintenance_window                    = "tue:04:29-tue:04:59"
   
+  /*Storage*/
```

Comment:
where is the relevant part about storage autoscaling?

Reply:
max_allocated_storage                 = var.max_allocated_storage

To enable Storage Autoscaling with instances that support the feature, we should pass the value ( higher than allocated_storage ) to above parameter. When we do that it automatically, enables autoscaling.

Reply:
got it